examples/hubspot/README.md

```diff
@@ -0,0 +1,41 @@
+# EveryAuth HubSpot Sample
+
+EveryAuth is the easiest way for your app to access HubSpot APIs. This is a sample Express application that demonstrates the use of EveryAuth in a hypothetical newsletter platform to connect to HubSpot and create new HubSpot Contact on newsletter signup. 
+
+Follow the [Integrating with HubSpot using Everyauth](https://fusebit.io/blog/everyauth-hubspot) blog post for detailed instructions.
+
+## Running the Sample App
+
+Make sure to follow the one-time [EveryAuth setup instructions](../README.md) first.
+
+Clone and build the repo:
+
+```bash
+git clone git@github.com:fusebit/everyauth-express.git
+cd everyauth-express
+npm i
+npm run build
+```
+
+Install sample dependencies:
+
+```bash
+cd examples/hubspot
+npm i
```

Comment:
Why npm install and build at the top level when only the subproject is used?

Reply:
I am pulling everyauth-express with `require("../..")` in the sample code - in other words the sample is pinned to the version of the code in the repo not in the npm.

Reply:
Regarding contents of readme, I thought Chris said it better not compete content-wise with the blog.

Reply:
Changed to fixed dep